common-requests/contribute.md

```diff
@@ -1,30 +1,35 @@
 ---
 layout: page
 title: "Contribute"
-description: "TBD"
+description: "Contribution process to help newcomers get started with Open Horizon"
 ---
 
-# Contribute
-
-## Process
-
 Contributions will only be considered when linked to an open issue, aligned with the project goals, and attested to by the Contributor (ex. signing off on a commit, accepting the [Developer Certificate of Origin](https://developercertificate.org/)).
 
-To get assigned to an issue, start with on open issue labelled `Good First Issue` in one of the repositories and submit a comment on the issue requesting that it be assigned to you.  To get a quick response, try "at" mentioning a Maintainer or Working Group lead in your comment.
+#### Get assigned
+To get assigned to an issue, start with on open issue labelled `Good First Issue` in one of the repositories and submit a comment on the issue requesting that it be assigned to you.  
```

Comment:
@diru1100 Not your fault, but there's a typo here.  "start with on open issue" should be "start with an open issue".